docs/timeseries/io.rst

```diff
@@ -58,22 +63,29 @@ Let's check that the time series has been read in correctly:
    plt.xlabel('Julian Date')
    plt.ylabel('SAP Flux (e-/s)')
 
-Reading other formats
+.. EXAMPLE END
+
+Reading Other Formats
 =====================
 
-At the moment only a few formats are defined in ``astropy`` itself, in part because
-there are not many well documented formats for storing time series. So in many
-cases, you will likely have to first read in your files using the more
+At the moment only a few formats are defined in ``astropy`` itself, in part
+because there are not many well-documented formats for storing time series. So
+in many cases, you will likely have to first read in your files using the more
 generic |Table| class (see :ref:`read_write_tables`). In fact, the
 :meth:`TimeSeries.read <astropy.timeseries.TimeSeries.read>` and
 :meth:`BinnedTimeSeries.read <astropy.timeseries.BinnedTimeSeries.read>` methods
-can do this behind the scenes - if the table cannot be read by any of the time
-series readers, these methods will try and use some of the default :class:`~astropy.table.Table`
-readers and then require users to specify the name of the important columns.
+can do this behind the scenes. If the table cannot be read by any of the time
+series readers, these methods will try to use some of the default
+:class:`~astropy.table.Table` readers and then require users to specify the
+names of the important columns.
 
-For example, if you are reading in a file called :download:`sampled.csv
-<sampled.csv>` where the time column is called ``Date`` and is an ISO string,
-you can do::
+Examples
+--------
+
+.. EXAMPLE START: Reading Other Formats for Storing TimeSeries
```

Comment:
"Other" is an unfortunate usage here I feel. It's not like we have kepler/tess and other, when most light curves are in fact in this other category so in fact kepler/tess is the special case. 

Maybe "generic"? I don't have a better suggestion (same comment applies for the section title, but it's problematic in the example gallery when a user tries to find out how to read in their light curve and they run into kepler/tess as a).

@astrofrog should comment on this.

Reply:
Maybe "Common Light Curve Formats"? If Kepler/Tess are the special cases?